en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -472,4 +473,75 @@ To create a For Each Loop:
+* When adding a new condition, the *On Error* condition field value is set to `None` by default.  Leaving this value set to `None` returns any errors that may occur in the *Watch* section and processes all steps inside this *On Error* section. 
+* When a condition is met in the *On Error* section, the Error Handler processes the conditions in sequential order and stops once the first condition met. After the first condition is met, any subsequent steps are skipped. If you want the process to continue, you can add an additional *On Error* section branch inside your existing *On Error* section.
+* When xref:ms_composer_test_flow.adoc[testing your flow], any errors caught by the Error Handler appear in the right pane of the action card. Successful steps are marked with a green check and failed steps are marked with a red X. 
+* When an error occurs in a watched action, the error information is saved as a data pill that can then be used for subsequent notification or record-updating actions. For example, if an error occurs, you can designate an action to send an email to your team containing the error information from the data pill.
+
+==== Steps 
+
+To add steps to the *On Error* section of your flow: 
+
+. <<watch-a-step,Select a step to watch>>.
+. In the *Error Handler* card, in the *On Error* section: 
+.. In the *Conditions to run this branch* drop-down, select one of the following conditions and then enter the requested criteria: 
+... `AND`: All conditions must be met.
+... `OR`: Any conditions can be met.
+... `None`: No conditions must be met and all errors are returned.
+... `Custom Logic`: <<use-custom-logic,Customize logic>> to designate how conditions will be met. 
+.. In the *After Running* drop-down, select: 
+... *Stop*: After processing all steps in the *On Error* section, errors are flagged and the flow stops. The *Status* field of the flow is marked as `FAILED` on the <<flow-details-page,*Run History*>> page.  
+... *Continue*: After processing all steps in the *On Error* section, errors are dismissed and the flow continues. If no other errors occur, the *Status* field of the flow is marked as `SUCCESS` on the <<flow-details-page,*Run History*>> page.  
+. If you want to add additional conditions to this *On Error* section, click *Add Condition*. 
+. If you want to look for a specific error condition and designate how it is handled within your existing *On Error* section, click *Add On Error* to add a new branch with steps. When adding a branch to the *On Error* section, in the *After Running* section, select: 
+.. *Continue*:  Processes *On Error* steps and then continues on the next step within the parent step. 
+.. *Propagate*:  Processes *On Error* steps and then pushes the error to the parent Error Handler section. The child step then stops processing and the Error Handler process returns to the parent step. 
```

Comment:
```suggestion
.. *Propagate*: Processes *On Error* steps and then pushes the error to the parent *Error Handler* section. The child step then stops processing and the error handler process returns to the parent step. 
```

Reply:
(Lowercase when referring generically, init cap/bold for UI.)

Reply:
Fixed these globally!